tests/bootstrap.php

```diff
@@ -1,5 +1,25 @@
 <?php
-
+/**
+ *
+ *
+ *
+ *
```

Comment:
Empty header header?

Reply:
Weird bug by the license checker! Fixed! :)